.travis.yml

```diff
@@ -3,16 +3,26 @@ language: go
 go:
   - 1.13.x
 
-cache:
-  directories:
-    - $GOPATH/pkg/mod
+matrix:
+  include:
+    - os: windows
+      dist: 1803-containers
+      env: VERSION=1.13.2 VARIANT=windows/windowsservercore-1803
+    - os: linux
+      env: VERSION=1.13.2 VARIANT=buster
 
-go_import_path: github.com/improbable-eng/kedge
+go_import_path: github.com/crufter/kedge
```

Comment:
should this be `github.com/improbable-eng/kedge`?

Reply:
Absolutely right, leftover from fork.